sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusClientBuilder.java

```diff
@@ -897,4 +913,162 @@ public ServiceBusReceiverClient buildClient() {
             return new ServiceBusReceiverClient(buildAsyncClient(), retryOptions.getTryTimeout());
         }
     }
+
+    /**
+     * Azure Service Bus queues and topic subscriptions provide a secondary subqueue, called a dead-letter queue (DLQ).
+     * The dead-letter queue doesn't need to be explicitly created and can't be deleted or otherwise managed
+     * independent of the main entity.
+     *
+     * This is builder for creating {@link ServiceBusReceiverClient} and {@link ServiceBusReceiverAsyncClient} to
```

Comment:
This documentation is incorrect.

Reply:
How do you want to change it ? The end of this sentence indicate `consume dead-letter messages from Service Bus entity` . That's what we do here, we create these client which point to dead letter queue.